tests/behat/bootstrap/BehatCliTrait.php

```
@@ -114,7 +112,7 @@ public function behatCliWriteFeatureContextFile(array $traits = []): string {
         else {
           // Not found in base namespace, let's check subdirectories
           // Get a list of directories under src/.
-          $base_dir = dirname(dirname(dirname(__DIR__))) . DIRECTORY_SEPARATOR . 'src';
+          $base_dir = dirname(__DIR__, 3) . DIRECTORY_SEPARATOR . 'src';
```

PHPMD (2.15.0)

The variable $base_dir is not named in camelCase. (undefined)

(CamelCaseVariableName)